sky/build/sdk_xcode_harness/FlutterApplication.xcodeproj/project.pbxproj

```diff
@@ -39,13 +41,15 @@
 			dstSubfolderSpec = 10;
 			files = (
 				9E07CFB61BE82D2600BCD8DE /* FlutterApplication.framework in Embed Frameworks */,
+				97950E421CDC05A500964A31 /* Flutter.framework in Embed Frameworks */,
 			);
 			name = "Embed Frameworks";
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		97F908AF1CDC02C500D4520F /* Flutter.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Flutter.framework; path = ../iphonesimulator/Flutter.framework; sourceTree = "<group>"; };
```

Comment:
Because the `iphonesimulator` path is hardcoded, this patch breaks devices builds. Fix incoming in the interpreter patches.